### PR TITLE
[pdq] README to refer to reference thresholds

### PR DESCRIPTION
**Pull request by Dcallies**
I've run into a number of platforms who aren't aware of the quality pass, and so think we should add it directly in the readme.

#accept2ship please

### CODE REVIEW

File: pdq/README.md [hunk 1]
```diff
@@ -2,19 +2,39 @@
...
+See also the [Meta Newsroom Post](https://newsroom.fb.com/news/2019/08/open-source-photo-video-matching) for context.
 
 As of November 2018 there are C++, PHP, and Java implementations.  Details are in the `*/README.md` files.
 
 ## Other Bindings and Implementations
 * Python: [faustomorales/pdqhash-python](https://github.com/faustomorales/pdqhash-python)
 * Rust: [darwinium-com/pdqhash](https://github.com/darwinium-com/pdqhash) - a great visualization of the algorithm also lives here.
 
+## Writing Your Own Hashing & Matching Implementations
+You can find a reference end-to-end hashing and matching solution in [facebook/ThreatExchange/python-threatexchange](https://github.com/facebook/ThreatExchange/tree/main/python-threatexchange). This tends to be the fatest way to install and play with the algorithm. You can see the implementation of the [PDQ SignalType for references for potential thresholds](https://github.com/facebook/ThreatExchange/blob/main/python-threatexchange/threatexchange/signal_type/pdq/signal.py)
```
Comment (BarrettOlson) [inline pdq/README.md:19]: s\fatest\fastest

File: pdq/README.md [hunk 2]
```diff
@@ -2,19 +2,39 @@
...
+
+### Hashing
+PDQ is heavily dependent on platform specific libraries to convert images into byte arrays. Even with the same exact images, you may get different PDQ hash values with different languages or underlying libraries. 
+
+We have generally considered implementations to be "correct" if, using images from the pdq/data directory
```
Comment (BarrettOlson) [inline pdq/README.md:24]: is this a both or either statement?

**Review by BarrettOlson — APPROVED** [review]
LGTM, one typo and nonblocking clarification question inline.

File: pdq/README.md (hunk 2 above)
Comment (Dcallies) [reply pdq/README.md:24]: Both

**Pull request merged**